src/components/Info.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="info-block">
+    {{ message }}
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
Consider always naming you component explicitly using "name" options.
This would be useful for debugging in Vue DevTools